soroban-sdk/src/address.rs

```diff
@@ -129,15 +129,15 @@ impl TryFromVal<Env, &Address> for RawVal {
 #[cfg(not(target_family = "wasm"))]
 impl TryFrom<&Address> for ScVal {
     type Error = ConversionError;
-    fn try_from(v: &Address) -> Result<Self, Self::Error> {
+    fn try_from(v: &Address) -> Result<Self, ConversionError> {
```

Comment:
What's the reason for all the changes from referencing the associated type to explicitly stating it? I don't see any functional change and so it seems like a matter of style. It looks like a large amount of the diff, so wondering why were going to that length.

Reply:
`Self::Error` is ambiguous between `TryFrom<_>::Error` and `ScVal::Error`, now that the latter is itself a defined symbol. Every change in the PR is just a fix to a compile error, nothing here is for style.